src/services/io/podio/JEventProcessorPODIO.cc

```diff
@@ -281,41 +281,47 @@ void JEventProcessorPODIO::Process(const std::shared_ptr<const JEvent> &event) {
     //            that are determined by hash, we have to ensure they are reproducible
     //            even if the collections are filled in unpredictable order (or not at
     //            all). See also below, at "TODO: NWB:".
```

Comment:
From CI, it doesn't look like any exceptions are triggered by missing collection id's. So we can delete this, perhaps?

Reply:
I am certainly open to making it a requirement that factories have to catch exceptions due to missing collections, instead of catching this in the event processor.

Reply:
I meant something like #1069